weave/schema.gql

```diff
@@ -0,0 +1,597 @@
+schema {
```

Comment:
Might be worth having a header here that comments that this is a hand-selected subset of the W&B GQL spec. And some notes about how/when to update

Reply:
done